test/distributed/org/apache/cassandra/distributed/impl/TracingUtil.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed.impl;
+
+import java.net.InetAddress;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+
+import org.apache.cassandra.db.ConsistencyLevel;
+
+
+/**
+ * Utilities for accessing the system_traces table from in-JVM dtests
+ */
+public class TracingUtil
+{
+    /**
+     * Represents an entry from system_traces
+     */
+    public static class TraceEntry
+    {
+        public final UUID sessionId;
+        public final UUID eventId;
+        public final String activity;
+        public final InetAddress source;
+        public final int sourceElapsed;
+        public final String thread;
+
+        private TraceEntry(UUID sessionId, UUID eventId, String activity, InetAddress sourceIP, int sourceElapsed, String thread)
+        {
+            this.sessionId = sessionId;
+            this.eventId = eventId;
+            this.activity = activity;
+            this.source = sourceIP;
+            this.sourceElapsed = sourceElapsed;
+            this.thread = thread;
+        }
+
+        static TraceEntry fromObjects(Object[] objects)
```

Comment:
The method name doesn't clearly convey the use of the method. Could we call this something different? This method converts a row result (list of deserialized objects) to a trace entry. Perhaps call this `fromRowResultObjects()`? I'm open to other name suggestions too.

Reply:
Renamed to `fromResultObjects` are requested.